src/backend/src/prisma/schema.prisma

```diff
@@ -109,6 +109,33 @@ enum Design_Review_Status {
   DONE
 }
 
+enum Graph_Type {
+  BAR
+  LINE
+  PIE
+}
+
+enum Data_Type {
```

Comment:
Lets name this something less vague like `Graph_Data_Unit` or something, idk what a good one would be